src/sidebar/panel.js

```diff
@@ -156,14 +156,23 @@ quill.on('text-change', () => {
 });
 
 const enableSync = document.getElementById('enable-sync');
-const giveFeedback = document.getElementById('give-feedback');
 const noteDiv = document.getElementById('sync-note');
 const syncNoteBody = document.getElementById('sync-note-dialog');
 const closeButton = document.getElementById('close-button');
 enableSync.textContent = browser.i18n.getMessage('syncNotes');
 syncNoteBody.textContent = browser.i18n.getMessage('syncNotReady2');
-giveFeedback.setAttribute('title', browser.i18n.getMessage('giveFeedback'));
-giveFeedback.setAttribute('href', SURVEY_PATH);
+
+const giveFeedback = document.getElementById('give-feedback');
+giveFeedback.textContent = browser.i18n.getMessage('feedback');
+giveFeedback.addEventListener('click', () => {
+  browser.tabs.create({
```

Comment:
Can we avoid adding "tabs" permission?

Reply:
For as simple as it seems, I couldn't get the `giveFeedback` element (`<li>`) to open a **new tab** in the **same window**. I was able to do the following, which none seem correct for opening the survey in terms of UX:

- same window, current tab (user's current tab gets overridden)
- new window w/ one tab (happened when `<a target="_blank">` was nested inside `<li>` element) (breaks user's workflow w/ having a new window open)

I may have missed something though so please let me know if you know of a better way.

Reply:
>  I couldn't get the giveFeedback element (<li>) to open a new tab in the same window.

Don't worry about changing this. the way the current feedback window opens is fine. We cannot introduce a new permission just for this.